In order to allow for installation of package for multiple versions of Python
Linux distribution often provide wrappers with correct shebangs tied to specific
Python implementation, e.g. Forthon-2.7, Forthon-python2.7, etc.
This patch fixes logic in Forthon_options to take that into account

Patch written by Kacper Kowalik <xarthisius@gentoo.org>
--- a/Lib/Forthon_options.py
+++ b/Lib/Forthon_options.py
@@ -92,7 +92,7 @@ class InputError(Exception):
 # --- Otherwise ignore the arguments. This is needed since for example this
 # --- module may be imported by the compilers module which is used by some
 # --- program other than Forthon.
-if os.path.basename(sys.argv[0]) == 'Forthon' or sys.argv[0] == '-c':
+if os.path.basename(sys.argv[0]).startswith('Forthon') or sys.argv[0] == '-c':
     (options, args) = parser.parse_args()
 else:
     (options, args) = parser.parse_args(args=[])
